test/testdata/deployednettemplates/recipes/geo-test-small/Makefile

```diff
@@ -0,0 +1,12 @@
+PARAMS=-w 80 -R 1 -N 4 -n 8 --node-template node.json --relay-template relay.json
```

Comment:
These recipe files do not belong in this PR, do they?

Reply:
I am using this to test algonet after these changes, but I will stash it before merging.

Reply:
ok - thanks.  please don't forget :)

Reply:
Removed